credoai/__init__.py

```diff
@@ -2,4 +2,14 @@
 Primary interface for Credo AI Lens package
 """
 
-__version__ = "1.1.2"
+from importlib.metadata import version
+
+# get package version programatically from installed wheel
+# does not rely on update to __init__.py for each new release
+# __version__ = version("credoai-lens")
+__version__ = "1.0.0"
```

Comment:
Why are you removing this version? This is the source of truth for what version the package is. `Setup.py` refers to this, as does pypi, through setup.

Reply:
The commented part was pattern matching based on deepchecks and doesn't work for us (hence commented)

The __version__ = "1.0.0" part was me testing to see that the warning would output if I imported the package locally.

Both were committed by mistake.